cmake/projects/libbacktrace/hunter.cmake

```diff
@@ -0,0 +1,42 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+#include(hunter_cmake_args)
+include(hunter_configuration_types)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    libbacktrace
+    VERSION
+    1.0.0-ca0de051
+    URL
+    "https://github.com/ianlancetaylor/libbacktrace/archive/ca0de0517f3be44fedf5a2c01cfaf6437d4cae68.tar.gz"
+    SHA1
+    7c7db4d096d96f329c453fb97b7a456fe823e943
+)
+
+#if (ANDROID OR IOS)
+# hunter_cmake_args(
+#    libbacktrace
+#    CMAKE_ARGS
+#    PKGCONFIG_EXPORT_TARGETS=libfoo;libbar
+#    DEPENDS_ON_PACKAGES=foo;bar
+#    EXTRA_FLAGS=--enable-bots
+# )
+#endif()
+
+hunter_configuration_types(libbacktrace CONFIGURATION_TYPES Release)
```

Comment:
Why hard code the configutation types. If debug is not supportet please add a comment about the reasons

Reply:
Because it uses the autotools scheme and this can only build one build type. If you remove this it will show an error:
```
[hunter ** FATAL ERROR **] Autotools PACKAGE_CONFIGURATION_TYPES has 2 elements: Release                                                                                                     
[hunter ** FATAL ERROR **] Debug. Only 1 is allowed                                                                                                                                          
[hunter ** FATAL ERROR **] [Directory:/source/hunter/_testing/Hunter/_Base/4ed7980/0fca6e7/2f6b703/Build/libbacktrace]                                                                       
                                                                                                                                                                                             
------------------------------ ERROR -----------------------------                                                                                                                           
    https://hunter.readthedocs.io/en/latest/reference/errors/autools.package.configuration.types.html                                                                                        
------------------------------------------------------------------                                                                                                                           
                                                                                                                                                                                             
CMake Error at /source/hunter/_testing/Hunter/_Base/Download/Hunter/unknown/4ed7980/Unpacked/cmake/modules/hunter_error_page.cmake:12 (message):                                             
Call Stack (most recent call first):                                                                                                                                                         
  /source/hunter/_testing/Hunter/_Base/Download/Hunter/unknown/4ed7980/Unpacked/cmake/modules/hunter_fatal_error.cmake:20 (hunter_error_page)                                                
  /source/hunter/_testing/Hunter/_Base/Download/Hunter/unknown/4ed7980/Unpacked/cmake/modules/hunter_autotools_configure_command.cmake:164 (hunter_fatal_error)                              
  /source/hunter/_testing/Hunter/_Base/Download/Hunter/unknown/4ed7980/Unpacked/cmake/modules/hunter_autotools_project.cmake:158 (hunter_autotools_configure_command)                        
  CMakeLists.txt:40 (hunter_autotools_project)
```